marvelous/series.py

```diff
@@ -42,7 +48,14 @@ def process_input(self, data, **kwargs):
             data["data"]["results"][0]["response"] = data
             data = data["data"]["results"][0]
 
+        # derive ID
         data["id"] = data["resourceURI"].split("/")[-1]
+        # derive thumbnail
+        thumb_dict = data.get("thumbnail", {})
+        if thumb_dict:
+            data["thumbnail"] = ".".join([thumb_dict["path"], thumb_dict["extension"]])
```

Comment:
May be clearer to use an f-string like` f"{thumb_dict["path]}.{thumb_dict["extension"]}" `instead of using a join(), but really is just a matter of preference.